Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackWidget.swift

```diff
@@ -50,6 +47,8 @@ struct SentryUserFeedbackWidget {
                     constraints.append(button.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor, constant: config.widgetConfig.layoutUIOffset.horizontal))
                 }
                 NSLayoutConstraint.activate(constraints)
+                
+                NotificationCenter.default.addObserver(self, selector: #selector(userCapturedScreenshot), name: UIApplication.userDidTakeScreenshotNotification, object: nil)
```

Comment:
`h`: I'm a bit confused. This fires when the user takes a screenshot, but we don't use the screenshot instead we take our own screenshot. Why do we need to subscribe to this?

Reply:
We can't access the real screenshot without pulling it from the photos library, which brings us to the same problem of plist key requirement that we're working around by doing this

Reply:
There are two flows as of now: 
1. the user opens the feedback form with the widget button
2. the user takes a screenshot, which we respond to by opening the form and affixing our version of the screenshot
I'll leave it to @bruno-garcia and @jas-kas to decide what they want to do with the UX, we can leave this to a later PR, but it shouldn't block this as this is more of a workaround to fix a potential issue with app store rejections.